### PR TITLE
Add parsimmon w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by rickyhopkins**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/parsimmon/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "parsimmon",
+  "description": "A monadic LL(infinity) parser combinator library",
+  "filename": "src/parsimmon.min.js",
+  "author": "Jeanine Adkisson <jneen at jneen dot net>",
+  "license": "MIT",
+  "repository": "https://github.com/jneen/parsimmon",
```
Comment (extend1994) [inline ajax/libs/parsimmon/package.json]: The format for cdnjs should be
```js
"repository": {
  "type": "git",
  "url": "https://github.com/jneen/parsimmon"
},
```

:)

File: ajax/libs/parsimmon/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "parsimmon",
+  "description": "A monadic LL(infinity) parser combinator library",
+  "filename": "src/parsimmon.min.js",
```
Comment (extend1994) [inline ajax/libs/parsimmon/package.json]: Main file for the browser with the relative path to `basePath` in auto-update config:
`parsimmon.umd.min.js`

File: ajax/libs/parsimmon/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "author": "Jeanine Adkisson <jneen at jneen dot net>",
+  "license": "MIT",
+  "repository": "https://github.com/jneen/parsimmon",
+  "keywords": [
+    "parsing",
+    "parse",
+    "parsers",
+    "parser combinators"
```
Comment (extend1994) [inline ajax/libs/parsimmon/package.json]: `combinators` is enough.

File: ajax/libs/parsimmon/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  ],
+  "npmName": "parsimmon",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```
Comment (extend1994) [inline ajax/libs/parsimmon/package.json]: Files for cdnjs is under `build`

File: ajax/libs/parsimmon/package.json (hunk 3 above)
Comment (rickyhopkins) [reply ajax/libs/parsimmon/package.json]: This was updated in my last commit, not sure how it didnt make it through to the pull request.

Comment (extend1994) [reply ajax/libs/parsimmon/package.json]: @rickyhopkins May I know your command history?

**Pull request merged**